spec/spec_helper.rb

```diff
@@ -0,0 +1,184 @@
+# frozen_string_literal: true
+
+require "simplecov"
+
+SimpleCov.add_filter "spec"
+SimpleCov.start
+
+require_relative "../lib/monthly_inventory"
+
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
I'd probably prefer not to have the commented-out boilerplate for rspec & rather have a section that is clearly listed as the 'default options' that we've left enabled vs. the stuff we've added